protocol/communities/community.go

```diff
@@ -1189,15 +1201,29 @@ func canDeleteMessageForEveryonePermissions() map[protobuf.CommunityMember_Roles
 	return roles
 }
 
+func (o *Community) GetAdminsAndTokenMasters() []*ecdsa.PublicKey {
+	return append(o.GetMemberTokenMasters(), o.GetMemberAdmins()...)
+}
+
 func (o *Community) GetMemberAdmins() []*ecdsa.PublicKey {
```

Comment:
I think it is better to name it
GetAdminMembers
The same for GetTokenMasterMembers